conf/application.conf

```diff
@@ -30,7 +43,8 @@ play.http.errorHandler = "uk.gov.hmrc.play.bootstrap.http.JsonErrorHandler"
 # Play Modules
 # ~~~~
 # Additional play modules can be added here
-
+play.modules.enabled += "config.DIModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
```

Comment:
Not here but can you change `uk.gov.hmrc.vatobligations.controllers.MicroserviceHelloWorld` to be `controllers.MicroserviceHelloWorld` please